eng/versioning/version_client.txt

```diff
@@ -103,27 +103,27 @@ com.azure:azure-spring-boot-starter-metrics;3.0.0-beta.1;3.0.0-beta.1
 com.azure:azure-spring-boot-starter-servicebus-jms;3.0.0-beta.1;3.0.0-beta.1
 com.azure:azure-spring-boot-starter;3.0.0-beta.1;3.0.0-beta.1
 com.azure:azure-spring-boot;3.0.0-beta.1;3.0.0-beta.1
-com.microsoft.azure:spring-cloud-azure-autoconfigure;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-context;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-eventhubs-stream-binder;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-messaging;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-servicebus-queue-stream-binder;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-servicebus-stream-binder-core;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-servicebus-topic-stream-binder;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-storage;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-stream-binder-test;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-azure-telemetry;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-starter-azure-eventhubs-kafka;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-starter-azure-eventhubs;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-starter-azure-servicebus;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-cloud-starter-azure-storage-queue;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-azure-core;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-azure-test;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-eventhubs;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-servicebus;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-integration-storage-queue;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-starter-azure-cache;1.2.8;1.3.0-beta.1
-com.microsoft.azure:spring-starter-azure-storage;1.2.8;1.3.0-beta.1
+com.azure:azure-spring-cloud-starter-autoconfigure;2.0.0-beta.1;2.0.0-beta.1
```

Comment:
Basically, if the original artifact does not contain the `starter`, we should not add them